cypress/integration/resources.spec.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import {
+  RESOURCE_CARD,
+  RESOURCE_SEARCH,
+  RESOURCE_SEARCH_BUTTON,
+  RESOURCE_RESET_BUTTON,
+} from '../../common/constants/testIDs';
+
+describe('resources', () => {
+  const CostSelect = 'input#react-select-paid-input';
+  const LanguageSelect = 'input#react-select-languages-input';
+  const CategorySelect = 'input#react-select-category-input';
+
+  beforeEach(() => {
+    cy.visitAndWaitFor('/resources/1');
+    cy.get('h1').should('have.text', 'Resources');
+  });
+
+  it('redirects on /resources to resources/1', () => {
+    cy.visitAndWaitFor('/resources');
+    cy.location('pathname').should('eq', '/resources/1');
+  });
+
+  it('loads many resource cards on resources/1', () => {
+    cy.visitAndWaitFor('/resources/1');
+    cy.findAllByTestId(RESOURCE_CARD).should('have.length', 20);
+  });
+
+  it('renders the second page of results when using pagination to visit page 2', () => {
+    cy.findByTestId('2').click();
+    cy.location('pathname').should('eq', '/resources/2');
+  });
+
+  it('renders the next page of results when using pagination to visit next page', () => {
+    cy.findByTestId('rightAngle').click();
```

Comment:
I feel like we can improve the testID name and also add it to the file for testIDs. Something like "NEXT_PAGE_BUTTON"

Reply:
all set, made them `NEXT_PAGE_BUTTON` and `PREV_PAGE_BUTTON` respectively and put into `testIds` dir.